lib/tasks/dev.rake

```diff
@@ -1,3 +1,95 @@
 desc "Fill the database tables with some sample data"
 task({ :sample_data => :environment }) do
+  p "Creating sample data"
+
+  if Rails.env.development?
+    Category.destroy_all
+    Listing.destroy_all
+    Message.destroy_all
+    Offer.destroy_all
+    User.destroy_all
+  end
+
+  12.times do
+    name = Faker::Name.first_name
+    username = Faker::Name.name
+    address = Faker::Address.full_address
+    bio = Faker::Quote.matz
+    location = Faker::Address.city
```

Comment:
If you would like to assign random account types by accessing the enums defined in your User model, you can do so by adding this line: ` account_types = User.account_types.keys.sample  # Generate random buyer or seller account types`